relay-kafka/src/config.rs

```diff
@@ -1,3 +1,11 @@
+//! Configuration primitives to configure the kafka producer and properly set up the connection.
+//!
+//! The configuration can be either;
+//! - [`TopicAssignment::Primary`] - the main and default kafka Configuration
+//! - [`TopicAssignment::Secondary`] - used to configure any additional kafka topic
```

Comment:
It seems we're using comas in these so:
```suggestion
//! - [`TopicAssignment::Primary`] - the main and default kafka Configuration,
//! - [`TopicAssignment::Secondary`] - used to configure any additional kafka topic,
```